src/cogent3/parse/gff.py

```diff
@@ -21,12 +21,11 @@
 
 
 def gff_parser(f):
-    if isinstance(f, str):
+    """delegates to the correct gff_parser based on the version"""
+    if isinstance(f, str) or isinstance(f, Path):
+        f = f if not isinstance(f, Path) else str(f)
```

Comment:
if you put the `f = f if not...` line before the `if isinstance` clause, you can remove the check for Path instance clause